proposals/001_Core_Contributors_governance.md

```diff
@@ -0,0 +1,66 @@
+# [PROPOSAL-001] Guru Network Core Contributors Incentives and Governance 
+
+## Summary
+
+This proposal outlines the incentive distribution plan for the Guru Network Core Contributors DAO, focusing on community engagement, core contributor rewards. The aim is to ensure transparency, accountability, and sustained growth for the Guru Network.
+
+## Background
+
+The Guru Network has consistently delivered high-impact products, including the Block Explorer, Data Warehouse, Flow, DexGuru V2, and Terminal integration, within a year. Our team, composed of seasoned Web3 professionals, has navigated both the highs and lows of the DeFi landscape. As we prepare for the next phase of growth, establishing a structured governance and incentive framework is crucial to maintaining our momentum and fostering a collaborative ecosystem.
+
+## Proposal
+
+### Incentive Distribution Plan
+
+To ensure the long-term prosperity of the Guru Network, we propose the following:
+
+1. **Create Core Contributors Seasons** and align them with Community Incentives Seasons in terms of periods and distributed allocation.
+2. **Setup a Core Contributors Governance Framework** tasked with developing and proposing a Community Governance Framework before the Guru Network Mainnet launch.
+
+Out of the total token supply, 60% is allocated for distribution as follows:
+
+- **20% for Guru Seasons 1-5 Community:**  
+  Rewarding Season Pass NFT holders and active community participants. The distribution is divided into seasons, with Seasons 1-2 are gaining and boosting rewards by participation now (Testnet phase). Activities snapshot is done close to the Mainnet launch. Community Seasons 3-5 planned for the next 24 months as a continuous community incentives aligned with Core Contributors Seasons.
```

Comment:
Activities snapshot is done close to the Mainnet launch

will be done

Reply:
done